app/jobs/index_finding_aid_job.rb

```diff
@@ -10,9 +10,26 @@ def perform(path, repo_id)
     stdout_and_stderr, process_status = Open3.capture2e(env, cmd)
 
     if process_status.success?
+      dest_path = "#{DulArclight.finding_aid_data}/xml/#{repo_id}"
+      FileUtils.mkdir_p(dest_path)
+      dest = "#{dest_path}/#{eadid_slug(path)}.xml"
+      FileUtils.copy_file(path, dest, preserve: true, dereference: true, remove_destination: true)
       puts stdout_and_stderr
     else
       raise DulArclight::IndexError, stdout_and_stderr
     end
   end
+
+  private
+
+  def eadid_slug(path)
+    basename = File.basename(path, ".*")
+    eadid_slug = nil
+    File.open(path, "r:UTF-8:UTF-8") do |f|
+      doc = Nokogiri::XML(f)
+      eadid_node = doc.at_xpath('/ead/eadheader/eadid')
```

Comment:
This could be `doc.xpath('normalize-space(/ead/eadheader/eadid')` to just give you the string.

Reply:
What does it return if the node does not exist?